### PR TITLE
Update cldr javascript library

### PR DESCRIPTION
**Pull request by PierreRambaud**
Update cldr library & apply cs fixer.

How to test:
Slider must works :)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/ps_facetedsearch/57)

### CODE REVIEW

File: _dev/cldr/number-formatter.js [hunk 1]
```diff
@@ -251,15 +252,20 @@ class CurrencyFormatter {
...
+      return formattedNumber
+        .split(CURRENCY_SYMBOL_PLACEHOLDER)
+        .join(this.numberSpecification.getCurrencySymbol());
+    }
+
+    return formattedNumber;
   }
 
   static build(specifications) {
     const symbol = new NumberSymbol(...specifications.symbol);
```
Comment (jolelievre) [inline _dev/cldr/number-formatter.js]: You can have a symbol even when it's not a price formatting?

File: _dev/cldr/number-formatter.js [hunk 2]
```diff
@@ -251,15 +252,20 @@ class CurrencyFormatter {
...
-    const currency = new CurrencyFormatter(
-      new PriceSpecification(
+    let specification;
+    if (specifications.currencySymbol) {
```
Comment (jolelievre) [inline _dev/cldr/number-formatter.js]: You don't need to check if it exists? Like with `specifications.currencySymbol != undefined` or `typeof specifications.currencySymbol != undefined` ?

File: _dev/cldr/number-formatter.js [hunk 3]
```diff
@@ -270,11 +276,24 @@ class CurrencyFormatter {
...
+        symbol,
+        parseInt(specifications.maxFractionDigits, 10),
+        parseInt(specifications.minFractionDigits, 10),
+        specifications.groupingUsed,
+        specifications.primaryGroupSize,
+        specifications.secondaryGroupSize,
+      );
+    }
+
+    const currency = new NumberFormatter(specification);
```
Comment (jolelievre) [inline _dev/cldr/number-formatter.js]: Maybe you should name it `currency` if it's for generic now

File: _dev/cldr/number-formatter.js (hunk 2 above)
Comment (PierreRambaud) [reply _dev/cldr/number-formatter.js]: I want to check for a value != null && != undefined

File: _dev/cldr/number-formatter.js (hunk 1 above)
Comment (PierreRambaud) [reply _dev/cldr/number-formatter.js]: Yep it's not a currency symbol, it's a list of symbol for the number format  https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Localization/Specification/NumberSymbolList.php

File: _dev/cldr/number-formatter.js (hunk 3 above)
Comment (PierreRambaud) [reply _dev/cldr/number-formatter.js]: you're right!

File: _dev/cldr/number-formatter.js (hunk 2 above)
Comment (jolelievre) [reply _dev/cldr/number-formatter.js]: Yes what I meant is if the `currencySymbol` property is not even present in `specifications`
Does this test work?

Comment (jolelievre) [reply _dev/cldr/number-formatter.js]: Ok I tested it it works in all use cases, fine for me!

**Comment by sarahdib** [comment]
@PierreRambaud 

the slider is missing 

Capture d’écran 2019-05-13 à 14 17 12

**Comment by PierreRambaud** [comment]
@sarahdib My bad, forget to says this PR needs to be merged first https://github.com/PrestaShop/PrestaShop/pull/13665

**Comment by PierreRambaud** [comment]
Should be ok now :)

**Comment by matks** [comment]
Thank you @PierreRambaud

**Pull request merged**